### ISSUE TITLE
Installing Contao 4.7 with Contao Manager gives green light with php 7.0.33 on debian 9.6

### ISSUE THREAD

**Issue by bmxp** [contao/contao-manager#405]
I tried for several hours to install contao 4.7 with contao-manager. I get a green light for PHP 7.0.33 and though everything is fine, just to get a failure on every installation effort.
Now I've digged deep into composer and packagagist just to find out that contao 4.7 needs at least PHP 7.1 and up. 
It would be have been very nice if I could have gotten a descriptive error what's wrong ...

**Comment by fritzmg** [comment]
What was the console output?

**Comment by aschempp** [comment]
Did you try to update or install? Because the Manager should not allow you to install (a fresh copy of) Contao 4.7 if you have PHP < 7.1

**Comment by bmxp** [comment]
@fritzmg I am sorry but I can not give a console output as I deleted everything from the machine.
It was however a recent and updated + dist-updated Debian 9.6 with PHP 7.0.33.
@aschempp It was a a fresh installation. Maybe I should have made a screnshot but contao manager 1.1.3 showed a green light for PHP 7.0.33.
Afterwards I tried to manually install composer and the install contao with composer. The version I was offered to install was 4.4.22 which seemed to me ok as it is the last version to support PHP < 7.1.
